index.html

```diff
@@ -40,5 +40,209 @@ <h1>I'm Benjamin Glad to see you!</h1>
             </div>
         </section>
     </div>
+    
+    <section class="middle-section-container">
+        <div class="image-area">
+          <img src="photo/logos/Snapshoot-Portfolio.png" alt="">
+        </div>
+
+      <div class="hero">
+          <div class="tonic">
+              <h1>Tonic</h1>
+            <div class="canopy-wrapper">
+              <p>CANOPY</p>
+              <ul>
+                <li>Back End Dev</li>
+                <li>2015</li>
+              </ul>
+            </div>
+          </div>
+
+          <div class="text-area">
+            <p>A daily selection of privately personalized reads; no accounts or sign-ups required.
+            </p>
+          </div>
+        </div>
+
+        <div class="middle-section-nav-bars">
+          <ul class="technology">
+            <li>html</li>
+            <li>css</li>
+            <li>javascript</li>
+          </ul>
+        </div>
+
+        <div class="project-link">
+          <h1>See Project</h1>
+        </div>
+    </section>
```

Comment:
- [ ] I think the card is not aligned the same as the Figma template  so please Fix the margin and padding to match the Figma design as much as possible, and also it is bordered with an orange border, kindly remove the border since it is not in the Figma design👌🏻👌🏻

- Your design 
![image](https://user-images.githubusercontent.com/58553711/177667809-49b7ee21-a004-4cfe-b144-267abe157bca.png)

- Figma design
![image](https://user-images.githubusercontent.com/58553711/177667879-6b7aace0-d5ad-4fd9-ab35-b16b68ef1258.png)